5.6/Dockerfile.rhel7

```diff
@@ -0,0 +1,61 @@
+FROM rhel7
```

Comment:
we're still going to build them in out of this repo because we do our automated testing via this repo, so that's how we valdiate the PR doesn't break the rhel image.

we don't have a good way to do automated testing once we're in dist-git/OSBS.